src/sentry_plugins/amazon_sqs/plugin.py

```diff
@@ -29,6 +29,22 @@ def get_regions():
     return public_region_list + cn_region_list
 
 
+def track_response_metric(fn):
+    # https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Queue.send_message
+    # boto3's send_message doesn't return success/fail or http codes
+    # success is a boolean based on whether there was an exception or not
+    def wrapper(*args, **kwargs):
+        try:
+            success = fn(*args, **kwargs)
+            metrics.incr("plugins.amazon-sqs.http_response", tags={"success": success})
```

Comment:
I still think moving `amazon-sqs` bit into a tag is better. @JTCunning 

So like, `metrics.incr(“data-forwarding.http_response”, tags={“plugin”: “amazon-sqs”, “result”: True})` instead will give you more flexibility when this inevitable is used elsewhere.

Reply:
I endorse this message.